### PR TITLE
Add Google Map through Maps API

### PR DESCRIPTION
**Pull request by raulcruise**
- Use a Google Map to showcase some of my favorite hometown spots
Instructions: https://github.com/raulcruise/my-portfolio/blob/master/walkthroughs/week-4-libraries/maps/maps-walkthrough.md

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -64,3 +64,239 @@ function clearComments() {
 function deleteComments() {
   fetch('/delete-data', {method: 'POST'}).then(() => getComments());
 }
+
+function createMap() {
+  // Create a map with a dark color scheme centered at my hometown, Anaheim
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:69]: Period at the end of the sentence.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -64,3 +64,239 @@ function clearComments() {
...
+  const map = new google.maps.Map(
+      document.getElementById('map'), {
+        center: {lat: 33.835982, lng: -117.914157},
+        zoom: 11,
+        styles: [
+          {elementType: 'geometry', stylers: [{color: '#292929'}]},
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:75]: I think it'd be nice to be consistent by having line breaks.

ex:
{
  elementType: ...,
  stylers: ...
}

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -64,3 +64,239 @@ function clearComments() {
...
+          {
+            featureType: 'water',
+            elementType: 'labels.text.stroke',
+            stylers: [{color: '#17263c'}]
+          }
+        ]
+      });
+
+  // Variables starting with shabu will be referring to "Hose of Shabu Shabu II"
+  const shabuContentString = '<div id="shabu-content">'+
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:157]: Let's have space before +

Reference:
https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations

Comment (jamesnam-google) [reply portfolio/src/main/webapp/script.js:157]: Here and below.

Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:157]: Consider creating some string variables that can help the lines below easier to follow.

ex:
const shabuShabuWikiUrl = ...
const koreanBarbequeueWikiUrl = ...

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -64,3 +64,239 @@ function clearComments() {
...
+    'you choose from a selection of menus which include a multitude '+
+    'of meats and vegetables and can order as much as you want! </p> '+
+    '</div>'+
+    '</div>';
+
+  // Variables starting with boiling will be referring to "Boiling Point"
+  const boilingContentString = '<div id="boiling-content">'+
+    '<h1 class="firstHeading">Boiling Point</h1>'+
+    '<div>'+
+    '<p><b>Boiling Point</b> is an establishment that specializes in <a href="https://en.wikipedia.org/wiki/Hot_pot">Taiwanese hot soup</a> cuisine. '+
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:173]: Same here and below when it comes to storing URL strings in variables.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -64,3 +64,207 @@ function clearComments() {
...
+      'you choose from a selection of menus which include a multitude ' +
+      'of meats and vegetables and can order as much as you want! </p> ' +
+      '</div>' +
+      '</div>';
+
+  // Variables starting with boiling will be referring to "Boiling Point".
+  const boilingContentString = '<div id="boiling-content">' +
+      '<h1 class="firstHeading">Boiling Point</h1>' +
+      '<div>' +
+      '<p><b>Boiling Point</b> is an establishment that specializes in <a href="' + hotPotWikiUrl + '">Taiwanese hot soup</a> cuisine. ' +
```
Comment (jamesnam-google) [inline portfolio/src/main/webapp/script.js:191]: Consider using line breaks - I think this line is kind of long.

Comment (raulcruise) [reply portfolio/src/main/webapp/script.js]: Done.

**Pull request merged**